src/utils.jl

```diff
@@ -175,27 +175,25 @@ function slurp(source)
     return final
 end
 
+getsource(source::Vector{UInt8}, ::Any) = source
```

Comment:
I was just typing this solution up as you pushed your commit.

Thank you for making this 100% cleaner and easier to read!! 🚀 💯